app/views/rsvp.erb

```diff
@@ -1,14 +1,34 @@
 <div class="offset" id="rsvp"></div>
+
 <% if session['access_token'] %>
-  <form class="form-horizontal" method="POST" action="/rsvp">
-  <fieldset>
+
+<% # to validate if the user in either of me and Nikhil's friend list
+    graph = Koala::Facebook::API.new(session['access_token'])
+    profile = graph.get_object('me')
+
+    #profile will be a JSON array , retrieve the name from there and display it
+
+    #get the friends list and see what all it displays  
+    friends = graph.get_connections('me','friends')
+    #print friends
+    #print friends.count
+    
+    #also we need to do a check for me and Nikhil, so that we can login and do anything
+    me_or_nikhil = (profile["id"] == ENV["debo_id"] || profile["id"] == ENV["nikhil_id"])?true:false
+    includes_value = friends.include?({"name"=>"Nikhil Bhardwaj","id"=>ENV["nikhil_id"]}) || friends.include?({"name"=>"Deboshrestha De","id"=>ENV["debo_id"]}) 
+%>
+  <%if includes_value || me_or_nikhil %>
```

Comment:
me_or_nikhil isn't required as we are be each others friends on FB :p
You can keep this for testing but this shouldn't be committed.